### PR TITLE
Remove user from user family tests written

### PR DESCRIPTION
**Pull request by Manik2708**
**What kind of change does this PR introduce?**

Increased the test code coverage to 100%

**Issue Number:**

Fixes #1820

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

File: src/resolvers/Mutation/removeUserFromUserFamily.ts [hunk 1]
```diff
@@ -110,6 +95,18 @@ export const removeUserFromUserFamily: MutationResolvers["removeUserFromUserFami
       );
     }
 
+    /*
+          userIsUserFamilyAdmin being true implies that the current user is an admin of userFamily.
+          If userIsUserFamilyAdmin is true pushes error message to errors list and breaks out of loop.
+        */
```
Comment (EshaanAgg) [inline src/resolvers/Mutation/removeUserFromUserFamily.ts:101]: Fix formatting.

File: tests/resolvers/Mutation/removeUserFromUserFamily.spec.ts [hunk 2]
```diff
@@ -54,12 +54,17 @@ describe("resolver -> Mutation -> removerUserFromUserFamily", () => {
       .spyOn(requestContext, "translate")
       .mockImplementation((message) => `Translated ${message}`);
     try {
+      if (testUserFamily == null) {
+        throw Error("Error in test");
+      }
```
Comment (EshaanAgg) [inline tests/resolvers/Mutation/removeUserFromUserFamily.spec.ts:59]: We don't need this. We can always assume that our helpers would return a valid `testUserFamily`

**Review by EshaanAgg — CHANGES_REQUESTED** [review]
You have used a lot of `eslint-disable` comments. Are they really necessary? Try to remove them.

File: tests/resolvers/Mutation/removeUserFromUserFamily.spec.ts (hunk 2 above)
Comment (Manik2708) [reply tests/resolvers/Mutation/removeUserFromUserFamily.spec.ts:59]: Then I have to use `es-lint ignore`. I was trying to get rid of these statements but unfortunately linter doesn't allow! I saw in a thread that someone is working on fixing this issue. But till that, ignore statements are the only way!

**Review by EshaanAgg — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**